client/src/pages/Contest/Contest.tsx

```diff
@@ -177,21 +197,54 @@ export default function Contest(): JSX.Element {
                   },
                 }}
               >
-                <Tab
-                  label={contestSubmissions.images ? `DESIGNS (${contestSubmissions.images.length})` : `DESIGNS (30)`}
-                />
+                {contestSubmissions.length && (
+                  <Tab label={contest && contest.images ? `DESIGNS (${contest.images.length})` : `DESIGNS (30)`} />
+                )}
                 <Tab label="BRIEF" />
               </Tabs>
             </ThemeProvider>
           </Toolbar>
           <Paper elevation={2}>
-            <Panel value={value} index={0}>
-              <SubmittedDesigns images={contestSubmissions.images} artist={contestSubmissions.artist} />
-            </Panel>
-            <Panel value={value} index={1}>
-              <Typography style={{ height: '200px', marginTop: '40px' }}>
-                {contestSubmissions.description ? contestSubmissions.description : 'Description of the contest'}
+            {contestSubmissions.length ? (
+              <Panel value={value} index={0}>
+                <div className={classes.imageWrapper}>
+                  <ImageList cols={4} gap={3} className={classes.imageList}>
+                    {contestSubmissions.map((submission) => (
+                      <SubmittedDesigns
+                        key={submission._id}
+                        images={submission.images}
+                        artistPic={submission.artistPic}
+                        artistName={submission.artistName}
+                      />
+                    ))}
+                  </ImageList>
+                </div>
+              </Panel>
+            ) : null}
+            <Panel value={value} index={contestSubmissions.length ? 1 : 0}>
+              <Typography align="center" variant="h3" className={classes.descriptionHeader}>
+                Tattoo Description:
+              </Typography>
+              <Typography align="center" variant="h5">
+                {contest ? contest.description : 'Description of the contest'}
               </Typography>
+              <ImageList cols={4} gap={10} className={classes.imageList}>
+                {contest &&
```

Comment:
You can write this in a single line using a ternary in Javascript object

Reply:
Done deal